addons/osio-addon/src/main/java/io/fabric8/launcher/osio/OsioMissionControl.java

```diff
@@ -122,6 +118,23 @@ public Boom launch(Projectile genericProjectile) throws IllegalArgumentException
     }
 
 
+    public OsioImportProjectile prepareImport(OsioImportProjectileContext context) {
+        final Space space = witClient.findSpaceById(context.getSpaceId());
+        Path path = gitSteps.clone(context);
+        for (ProjectilePreparer preparer : preparers) {
```

Comment:
We might have to make sure that the preparers are robust enough to handle imports? (For example, what if the imported code doesn't have a README.md? Will the preparer fail, ignore it or create a README?)

Reply:
They are, the ProjectileContext type is different. In your example, it will ignore